sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -466,7 +466,7 @@ class Handle(DictMixin):
      the handle has last been reconnected. (UTC)
     :keyword ~datetime.datetime last_reconnect_time: Time handle was last connected to (UTC)
     :keyword access_rights: Access rights of the handle.
-    :paramtype access_rights: List[Literal['Read', 'Write', 'Delete']]
+    :paramtype access_rights: Optional[List[Literal['Read', 'Write', 'Delete']]]
```

Comment:
Strange that this is needed since it's a keyword. All the other keywords here are not marked with `Optional` but API View has them all as Options...  :shrug: